common/app/model/Formats.scala

```diff
@@ -146,8 +147,31 @@ object PressedContentFormat {
     }
   }
 
-  implicit val sectionIdFormat = Json.format[SectionId]
-  implicit val pillarFormat = Pillar.format
+  implicit val designTypeFormat: Format[DesignType] = new Format[DesignType] {
+    override def reads(json: JsValue): JsResult[DesignType] = json match {
+      case JsString("Article") => JsSuccess(com.gu.contentapi.client.utils.Article)
+      case JsString("Immersive") => JsSuccess(com.gu.contentapi.client.utils.Immersive)
+      case JsString("Media") => JsSuccess(com.gu.contentapi.client.utils.Media)
+      case JsString("Review") => JsSuccess(com.gu.contentapi.client.utils.Review)
+      case JsString("Analysis") => JsSuccess(com.gu.contentapi.client.utils.Analysis)
+      case JsString("Comment") => JsSuccess(com.gu.contentapi.client.utils.Comment)
+      case JsString("Feature") => JsSuccess(com.gu.contentapi.client.utils.Feature)
+      case JsString("Live") => JsSuccess(com.gu.contentapi.client.utils.Live)
+      case _ => JsError(s"Unknown design type: '$json'")
+    }
+    override def writes(dt: DesignType): JsValue = dt match {
+      case com.gu.contentapi.client.utils.Article => JsString("Article")
+      case com.gu.contentapi.client.utils.Immersive => JsString("Immersive")
+      case com.gu.contentapi.client.utils.Media => JsString("Media")
+      case com.gu.contentapi.client.utils.Review => JsString("Review")
+      case com.gu.contentapi.client.utils.Analysis => JsString("Analysis")
+      case com.gu.contentapi.client.utils.Comment => JsString("Comment")
+      case com.gu.contentapi.client.utils.Feature => JsString("Feature")
+      case com.gu.contentapi.client.utils.Live => JsString("Live")
+    }
+  }
```

Comment:
that's a bit of boilerplate but the other solution would have been to add a new dependency like https://github.com/xdotai/play-json-extensions to automatically handle the formatting of sealed trait object.
Given the design types shouldn't change very often I prefer to go for those 20 lines of codes rather than adding another dependency.